packages/components/tsconfig.json

```diff
@@ -2,11 +2,8 @@
 	"extends": "../../tsconfig.base.json",
 	"compilerOptions": {
 		"rootDir": "src",
-		"declarationDir": "build-types",
+		"declarationDir": "build-types"
 	},
-	"references": [],
-    "include": [
-		"src/dashicon/*",
-		"src/tip/*"
-	],
+	"references": [ { "path": "../primitives" } ],
```

Comment:
We have to add references so TypeScript can figure out the build order properly. It’s the one “gotcha” to the setup

We would want this for all the monorepo packages this package depends on, but we have a problem where this isn't a "leaf" package so we need to add them as we need them.

This should fix the test issues that appeared on this branch.

Reply:
We need to add one for `@wordpress/element` as well. We could add all the packages that have opted-in to typing and appear in this package`s package.json.